src/lib.rs

```diff
@@ -98,15 +98,31 @@ pub mod author {
 /// This module defines the `Commit` data structure together with its utility
 /// enum `CommitParseError`.
 pub mod commit {
+    /// The set of errors which may occur.
     #[derive(Debug)]
     pub enum CommitParseError {
-        CommitMissing,
-        AuthorMissing,
-        DateMissing,
+        /// Parsing the author information was not possible.
         AuthorFailed(crate::author::AuthorParseError),
+
+        /// There were no author information.
+        AuthorMissing,
+
+        /// There was no commit.
```

Comment:
Hmm, maybe commit hash might be clearer?

Reply:
Okay. :wrench:

Reply:
:wrench: :heavy_check_mark: